src/io/flutter/dart/FlutterDartAnalysisServer.java

```diff
@@ -88,4 +107,25 @@ private void processNotification(JsonObject response) {
       }
     }
   }
+
+  private class RefreshDartAnalysisServerServiceExThread extends Thread {
+    RefreshDartAnalysisServerServiceExThread() {
+      setDaemon(true);
+    }
+
+    @Override
+    public void run() {
+      //noinspection InfiniteLoopStatement
+      while (true) {
```

Comment:
I think we should prefer using something like JobScheduler.getScheduler().scheduleWithFixedDelay() as it may play better with the IntelliJ APIs.

https://github.com/flutter/flutter-intellij/blob/master/src/io/flutter/sdk/FlutterSdkManager.java#L45

Reply:
Ah, this is much better. Thanks.